spec/validators/efl_year_validator_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+RSpec.describe EflYearValidator do
+  let(:award_year) { nil }
+
+  before do
+    %w[IeltsForm OtherEflQualificationForm ToeflForm].map do |form|
+      stub_const(form.camelcase.to_s, Class.new).class_eval do
+        include ActiveModel::Validations
+        include ActiveModel::Model
+
+        attr_accessor :award_year
+
+        validates :award_year, efl_year: true
+      end
+    end
+  end
+
+  describe 'is valid' do
+    %w[IeltsForm OtherEflQualificationForm ToeflForm].map do |model|
+      context 'when year is nil' do
+        it 'returns no error' do
+          expect(model.constantize.new(award_year: award_year)).to be_valid
+        end
+      end
+
+      context 'when year is in the correct format' do
+        let(:award_year) { Time.zone.now.year.to_s }
+
+        it 'returns no error' do
+          expect(model.constantize.new(award_year: award_year)).to be_valid
+        end
+      end
+
+      context 'when year is valid for all forms' do
+        let(:award_year) { '1981' }
```

Comment:
Can you test that any leading and trailing whitespace is ignored?

```suggestion
        let(:award_year) { ' 1981 ' }
```